src/main/java/ml/shifu/shifu/core/processor/stats/MapReducerStatsWorker.java

```diff
@@ -176,6 +180,9 @@ public boolean doStats() throws Exception {
             processor.syncDataToHdfs(modelConfig.getDataSet().getSource());
         }
 
+        //run daily stat compute
+        updateDailyStatWithMRJob();
```

Comment:
this seems a issue, if dateColumnName not specified, such job should not be called